sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -30,7 +32,32 @@ export interface AtomXmlSerializer {
   // eslint-disable-next-line @typescript-eslint/ban-types
   serialize(requestBodyInJson: object): Record<string, unknown>;
 
-  deserialize(response: HttpOperationResponse): Promise<HttpOperationResponse>;
+  deserialize(response: FullOperationResponse): Promise<FullOperationResponse>;
+}
+
+function prepare(
```

Comment:
nit: rename to `applyRequestOptions`?

Also, it updates the request object in place, should it return void instead?

Reply:
done!